python/lsst/ts/watcher/rules/mtm2_force_error.py

```diff
@@ -0,0 +1,193 @@
+
+    @classmethod
+    def get_schema(cls) -> dict[str, typing.Any]:
+        schema_yaml = """
+            $schema: 'http://json-schema.org/draft-07/schema#'
+            description: Configuration for MTM2ForceError rule.
+            type: object
+            properties:
+                force_error_axial:
+                    description: >-
+                        Limit of the force error of axial actuator (N).
+                    type: number
+                    default: 5.0
+                force_error_tangent:
+                    description: >-
+                        Limit of the force error of tangent link (N).
+                    type: number
+                    default: 10.0
+                poll_interval:
+                    description: Time delay between polling updates (second).
+                    type: number
+                    default: 1.0
+
+            required:
+            - force_error_axial
+            - force_error_tangent
+            - poll_interval
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def setup(self, model) -> None:
+
+        self._remote = model.remotes[("MTM2", 0)]
+
+    def compute_alarm_severity(self) -> AlarmSeverityReasonType:
+        """Compute and set alarm severity and reason.
+
+        Returns
+        -------
+        None, if no change or unknown, or a tuple of two values:
+
+        severity: `lsst.ts.xml.enums.Watcher.AlarmSeverity`
+            The new alarm severity.
+        reason : `str`
+            Detailed reason for the severity, e.g. a string describing
+            what value is out of range, and what the range is.
+            If ``severity`` is ``NONE`` then this value is ignored (but still
+            required) and the old reason is retained until the alarm is reset
+            to ``nominal`` state.
+
+        Notes
+        -----
+        You may return `NoneNoReason` if the alarm state is ``NONE``.
+        """
+
+        assert self._remote is not None
+
+        list_axial = list()
+        if self._remote.tel_axialForce.has_data:
+            list_axial = self._check_out_of_range(
+                self._remote.tel_axialForce.get(), self.config.force_error_axial
+            )
+
+        list_tangent = list()
+        if self._remote.tel_tangentForce.has_data:
+            list_tangent = self._check_out_of_range(
+                self._remote.tel_tangentForce.get(), self.config.force_error_tangent
+            )
+
+        if (len(list_axial) != 0) and (len(list_tangent) != 0):
+            return (
+                AlarmSeverity.SERIOUS,
+                f"Axial ({list_axial}) and tangent ({list_tangent}) force errors out of normal range.",
+            )
+
+        elif len(list_axial) != 0:
+            return (
+                AlarmSeverity.SERIOUS,
+                f"Axial ({list_axial}) force error out of normal range.",
+            )
+
+        elif len(list_tangent) != 0:
```

Comment:
same as above.

Reply:
Fixed in the commit: [e5e703b](https://github.com/lsst-ts/ts_watcher/pull/83/commits/e5e703b275b024b0fd34185ef4cae12882298a75). Thanks!